app/assets/javascripts/discourse/app/components/composer-editor.js

```diff
@@ -634,12 +641,34 @@ export default Component.extend({
 
     const $element = $(this.element);
 
+    $.blueimp.fileupload.prototype.processActions = uploadProcessorActions;
```

Comment:
This doesn't feel right to me. `$.blueimp` is a vendor concern that our Discourse code base should not have to be aware of.

Reply:
> This doesn't feel right to me. `$.blueimp` is a vendor concern that our Discourse code base should not have to be aware of.

That is their only documented way of registering `processActions` callbacks. That being said, I'm working on merging the theme-component that uses this into core and will look into removing this!